python/llm/src/bigdl/llm/vllm/model_executor/models/bigdl_chatglm.py

```diff
@@ -58,23 +58,26 @@ def __init__(
         config,
         device: Optional[str] = None,
         max_model_len: Optional[int] = None,
+        load_in_low_bit: str = 'sym_int4'
     ):
         super().__init__(config, device, max_model_len)
         self.config = config
         # TODO(gc): later change this to a switch?
-        if True:
-            from bigdl.llm.transformers import AutoModelForCausalLM
-            from bigdl.llm import optimize_model
+        from bigdl.llm.transformers import AutoModelForCausalLM
+        torch_dtype = 'auto'
 
-        # low_bit = 'sym_int4'
+        if load_in_low_bit == 'bf16':
```

Comment:
Since modified GPU example, should we add fp16 check here?
```
model = AutoModelForCausalLM.from_pretrained(model_path, trust_remote_code=True,
                                                     use_cache=True,
                                                     load_in_low_bit="fp16",
                                                     torch_dtype=torch.float16)
```